### ISSUE TITLE
Compilation issue -'array' - constant variable cannot be passed as reference

### ISSUE THREAD

**Issue by damiangr** [EA31337/EA31337-classes#661]
Hi , i've tried to compile v2.010 version(with MT5 ver 5 build 3446) and it gives me following errors: 

```
'array' - constant variable cannot be passed as reference	Array.mqh	777	39
   in template 'void ArrayPushObject(X&[],X&)' specified with [X=IValueStorage*]	Array.mqh	776	6
'array' - constant variable cannot be passed as reference	Array.mqh	778	26
   in template 'void ArrayPushObject(X&[],X&)' specified with [X=IValueStorage*]	Array.mqh	776	6
```

**Comment by kenorb** [comment]
> For a quick fix:
> 
> in `EA31337-classes/Array.mqh` line 756
> 
> search for a static int ArraySize(const ARRAY_REF(X, array)) { and remove const word.

**Comment by kenorb** [comment]
Fixed in [v2.011.1](https://github.com/EA31337/EA31337-classes/releases/tag/v2.011.1) and [v2.012.1](https://github.com/EA31337/EA31337-classes/releases/tag/v2.012.1). And it's going to be fixed in the future releases.
